cms/src/test/resources/test-data/api-specifications/ToPrettyJsonStringDeserializerTest/jsonObjectWithJsonValueContainingXmlPrettyPrinted.json

```diff
@@ -0,0 +1,14 @@
+{
+  "resourceType": "MedicationRequest",
+  "text": {
+      "status": "generated",
+      "div": "<div xmlns=\\"http://www.w3.org/1999/xhtml\\"> <p> <b>Generated Narrative with Details</b> </p> <p> <b>id</b>: medrx002 </p> <p> <b>identifier</b>: 12345 (OFFICIAL) </p> <p> <b>status</b>: active </p> <p> <b>intent</b>: order </p> <p> <b>medication</b>: <a>prescribed medication</a> </p> <p> <b>subject</b>: <a>Donald Duck</a> </p> <p> <b>context</b>: <a>encounter that leads to this prescription</a> </p> <p> <b>authoredOn</b>: 01/03/2015 </p> <h3>Requesters</h3> <table> <tr> <td>-</td> <td> <b>Agent</b> </td> <td> <b>OnBehalfOf</b> </td> </tr> <tr> <td>*</td> <td> <a>Patrick Pump</a> </td> <td> <a>Organization/f002</a> </td> </tr> </table> <p> <b>reasonCode</b>: Essential hypertension (disorder) <span>(Details : {SNOMED CT code '59621000' = 'Essential hypertension', given as 'Essential hypertension (disorder)'})</span> </p> <p> <b>dosageInstruction</b>: </p> </div>"
```

Comment:
i dont think you should have double \'s in the string-encoded div block here. looks like it's breaking the json unless i'm missing something